src/main/java/com/solvd/laba/carina/products/gui/pages/ProductDetailPage.java

```diff
@@ -0,0 +1,28 @@
+package com.solvd.laba.carina.products.gui.pages;
+
+import com.zebrunner.carina.webdriver.decorator.ExtendedWebElement;
+import com.zebrunner.carina.webdriver.decorator.PageOpeningStrategy;
+import com.zebrunner.carina.webdriver.gui.AbstractPage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.lang.invoke.MethodHandles;
+
+public class ProductDetailPage extends AbstractPage {
+
+    @FindBy(xpath = "//div[@class='product-details']")
+    private ExtendedWebElement productDetails;
```

Comment:
It displays the details of a product on this page. On the products page, if you click on the view product button of a product from the list, it will lead you to a new page and display the details of this product.

Reply:
Understand. I have verified the product title, category, price, availability, condition, branch, and add to cart.